### PR TITLE
completed css-fonts-and-colors-Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**
Screenshot 2024-10-29 at 3 19 40 PM

### CODE REVIEW

File: css-fonts-and-colors/styles.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Nunito, sans-serif;
+  font-optical-sizing: auto;
+  font-weight: <weight>;
```
Comment (shawnkost) [inline css-fonts-and-colors/styles.css]: This should be a number. If this is for normal text, then it should be 400.

File: css-fonts-and-colors/styles.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  padding: 1.5rem 0;
+  background-color: lightblue;
+}
+
+h2 {
+  margin-bottom: 3.5rem;
+  font-size: 2rem;
+  text-align: center;
+  font-family: Ubuntu, sans-serif;
+  font-weight: 500;
```
Comment (shawnkost) [inline css-fonts-and-colors/styles.css]: Headings should be at least 600 or 700 so that it's bold

**Review by shawnkost — APPROVED** [review]
Approved, but left some comments

File: css-fonts-and-colors/styles.css (hunk 2 above)
Comment (GCarolyne) [reply css-fonts-and-colors/styles.css]: Thank you I changed it.

**Pull request merged**